node/clients/checkout.ts

```diff
@@ -27,6 +27,9 @@ export class Checkout extends JanusClient {
         ...(ctx.storeUserAuthToken
           ? { VtexIdclientAutCookie: ctx.storeUserAuthToken }
           : null),
+        ...ctx.adminUserAuthToken
+          ? { VtexIdClientAutCookie: ctx.adminUserAuthToken }
+          : null,
```

Comment:
you'll need to create a separate client to include this token, as this can have unintended side-effects. see vtex-apps/checkout-graphql#61

Reply:
remember to remove the token in this client 👀